src/Microsoft.AspNet.Diagnostics/Views/BaseView.cs

```diff
@@ -55,7 +55,7 @@ public abstract class BaseView
             Output = new StreamWriter(Response.Body);
```

Comment:
I tried the `AutoFlush=true` but didn't work as its supposed to probably because of the following?(BTW I tried this with `ErrorPageSample` in the solution):

https://msdn.microsoft.com/en-us/library/system.io.streamwriter.autoflush(v=vs.110).aspx

"Flushing the stream will not flush its underlying encoder unless you explicitly call Flush or Close. Setting AutoFlush to true means that data will be flushed from the buffer to the stream after each write operation, but the encoder state will not be flushed. This allows the encoder to keep its state (partial characters) so that it can encode the next block of characters correctly. This scenario affects UTF8 and UTF7 where certain characters can only be encoded after the encoder receives the adjacent character or characters."